client/state/index.js

```diff
@@ -39,10 +38,23 @@ const reducer = combineReducers( {
 	ui
 } );
 
-var createStoreWithMiddleware = applyMiddleware(
-	thunkMiddleware,
-	analyticsMiddleware
-);
+let middleware = [ thunkMiddleware ];
+
+if ( typeof window === 'object' ) {
+	middleware = [
+		...middleware,
+		require( './themes/middlewares.js' ).analyticsMiddleware
+	];
+}
+
+let createStoreWithMiddleware = applyMiddleware.apply( null, middleware );
+
+function getInitialState() {
```

Comment:
And maybe add a comment here that this is for bootstrapping, using the initialState passed from the server (maybe even with a note in which file that is set?)